sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -46,6 +47,15 @@ static void init(SentryOptions options, Context context, ILogger logger) {
     }
   }
 
+  private static void setDefaultInApp(Context context, SentryOptions options) {
+    // If context passed in is an application-defined class, we can use to white-list frames
+    String packageName = context.getClass().getPackage().getName();
```

Comment:
why's that? why do we want to white-list the android-package?

Reply:
the user could hide the MainApplication whatever they want to, like `com.my.company.mythings.myusefulthings.idontknowwhat`.
not sure if it's a good idea, I'd prefer to keep opt-in by the user.
Or to use the package name like `packageInfo.packageName` which is supposed to be way more accurate

Reply:
OptIn is the worst experience. We want to give sensible defaults.
If picking up from packageInfo.packageName is the best approach then lets do that!